### PR TITLE
Translation to Portuguese (Portugal)

### PR DESCRIPTION
**Pull request by glxxyxafonso**
After seeing all of the languages available, i've talked to my Robotics teacher and we are looking to do a photo booth for a special day!

As we are a portuguese school, i've decided to create this translation. Hope you guys enjoy and accept it!

### CODE REVIEW

**Comment by glxxyxafonso** [comment]
Still waiting for a response.

**Comment by werdeil** [comment]
Sorry @glxxyxafonso we didn't see your PR, we will review it and I guess it will be merged soon.

Thanks a lot !

File: pibooth/language.py [hunk 1]
```diff
@@ -159,6 +159,23 @@
         'finished': "Takk",
         'oops': "Ops, noe gikk galt",
     },
+        },
+    'pt': {
```
Comment (glxxyxafonso) [inline pibooth/language.py:163]: pt is also the biggest gTLD

**Review by werdeil — APPROVED** [review]

**Comment by glxxyxafonso** [comment]
@werdeil Nice! Portugal is here!

**Pull request merged**